examples/sample-hono-stripe/README.md

```
@@ -0,0 +1,122 @@
+stripe listen --forward-to localhost:3000/webhook
+```
+
+Trigger a test event:
+
+```bash
+stripe trigger payment_intent.succeeded
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)